### ISSUE TITLE
Error after entering API token

### ISSUE THREAD

**Issue by c6y** [precipice/campfire_export#21]
"Unable to export account: can't modify frozen Fixnum"

Thanks for any help! ~ Kai

**Comment by c6y** [comment]
works with Ruby 1.9.3 – used rvm to install it along Ruby 2.0.0

**Comment by precipice** [comment]
Yup, sorry about that. I haven't updated to 2.0 yet. I'll add a README note since others have hit this.

Thanks for the report. 

-M
